perun-core/src/test/java/cz/metacentrum/perun/core/entry/GroupStructureSynchronizationIntegrationTest.java

```diff
@@ -447,6 +454,104 @@ public void removeAllGroupsTest() throws Exception {
 		assertTrue("List of sub groups of base group should be empty!", subGroups.isEmpty());
 	}
 
+	@Test
+	public void additionalStringAttributeIsSet() throws Exception {
+        System.out.println(CLASS_NAME + "additionalStringAttributeIsSet");
+
+		AttributeDefinition loginAttrDef = setGroupAttribute(ADDITIONAL_STRING);
```

Comment:
You probably want to change name of this variable. It is a little confusing now.

Reply:
Yes, I do. I have changed it.